src/wirecraft/client/game.py

```diff
@@ -319,3 +330,14 @@ def show_inventory(self):
                 "You have 10 switches",
             )
         )
+    def show_tasks(self):
+        """Task button action."""
+        # display a window on the left side taking all the height of the screen
```

Comment:
I don't know if this is intended, but task window don't take all the height of the screen